openpaas-james/mailbox/encrypted/memory/src/main/java/com/linagora/openpaas/encrypted/InMemoryEncryptedMessageManager.scala

```diff
@@ -40,28 +42,37 @@ class InMemoryEncryptedMessageManager @Inject()(messageManager: MessageManager,
       .collectSeq()
       .block()
 
-    val out: ByteArrayOutputStream = new ByteArrayOutputStream
-    Encrypter.forKeys(keys.map(key => key.payload).asJava).encrypt(ByteSource.wrap(msgIn.readAllBytes()), out)
+    val out: FileBackedOutputStream = new FileBackedOutputStream(FILE_THRESHOLD)
+    try {
+      Encrypter.forKeys(keys.map(key => key.payload).asJava)
+        .encrypt(ByteSource.wrap(msgIn.readAllBytes()), out)
```

Comment:
Mmmm no.

We should use `encrypt(MimeMessage xxx)` version for it to be inter-operable with enygmail.